src/js/login/containers/Main.jsx

```diff
@@ -141,7 +145,7 @@ class Main extends React.Component {
 const mapStateToProps = (state) => {
   const userState = state.user;
   return {
-    login: userState.login,
+    loginUrl: userState.loginUrl,
```

Comment:
I think naming of these fields is confusing.  I'm going to either
1)  rename `login.loginUrl` to `login.loginUrls` so it's clear when we're talking about the object (which holds 2 URLS, one for login and one for verification) and the actual loginURL (the 'first' property),
or
2) just make `verifyUrl` its own prop and get rid of the `loginUrl object; this will require a separate action to update the `verifyUrl` but might be clearer, since we have a separate `logoutUrl` prop already

LMK if you have opinions about one or the other